composer.json

```diff
@@ -13,11 +13,11 @@
 	],
 	"require": {
 		"php": "^8.0 || ^7.4",
-		"microsoft/kiota-authentication-phpleague": "^0.4.0 || ^0.5.0",
-		"microsoft/kiota-http-guzzle": "^0.4.0",
-		"microsoft/kiota-serialization-json": "^0.3.0",
-		"microsoft/kiota-serialization-text": "^0.3.0",
-		"microsoft/kiota-abstractions": "^0.4.0",
+		"microsoft/kiota-authentication-phpleague": "dev-dev",
+		"microsoft/kiota-http-guzzle": "dev-dev",
+		"microsoft/kiota-serialization-json": "dev-dev",
+		"microsoft/kiota-serialization-text": "dev-dev",
```

Comment:
Changed to `dev`  to easily accommodate further changes. We can bump versions after static analysis etc passes on these dependencies